lighthouse-core/scripts/compare-timings.js

```diff
@@ -11,6 +11,8 @@
 //     node lighthouse-core/scripts/compare-timings.js --name base --name pr --compare
```

Comment:
Are you interested in comments that combines the intended use case of this script with your new one?

Reply:
i am interested in that in 8 days from now. :)